webpack/components/TemplateSyncResult/components/SyncResultList.js

```diff
@@ -19,11 +19,9 @@ const SyncResultList = ({ pagination, pageChange, templates, editPaths }) => (
       />
     ))}
     <Pagination
-      viewType="list"
       itemCount={templates.length}
       pagination={pagination}
```

Comment:
I believe this option is unsupported we now have directly perPage, perPageOptions and so...

Reply:
👍  extracted the props from it